disco_aws_automation/disco_vpc_peerings.py

```diff
@@ -213,6 +213,20 @@ def _create_peering_routes(self, peerings):
                 str(source_network.network_cidr)
             )
 
+    def _delete_peering_routes(self, peering):
+        """Delete routes related to a peering connection"""
+        route_tables = self._get_peering_route_tables(peering['VpcPeeringConnectionId'])
+        for route_table in route_tables:
+            route_table_id = route_table['RouteTableId']
+            peering_routes = [route for route in route_table['Routes']
+                              if route.get('VpcPeeringConnectionId') == peering['VpcPeeringConnectionId']]
+            if peering_routes:
+                for route in peering_routes:
+                    self.client.delete_route(
```

Comment:
Not sure if this is possible in this case but you might be able to use `DiscoSubnet.delete_route`